packages/extractor-vue/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { vueScriptExtractor } from "./vue-script-extractor"
+import { vueScriptSetupExtractor } from "./vue-script-setup-extractor"
+import { vueTemplateExtractor } from "./vue-template-extractor"
+
+export const vueExtractors = [
+  vueScriptExtractor,
+  vueScriptSetupExtractor,
+  vueTemplateExtractor,
+]
```

Comment:
this way you will parse vue files with vue-sfc-compiler 3 times...

Reply:
> this way you will parse vue files with vue-sfc-compiler 3 times...

Happy for you to take a stab on this. Not sure how to make it work properly with sourcemaps and all by merging handling of all Vue SFC parts into one extractor.